src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/authorization/ReferralAccessFilter.kt

```diff
@@ -6,15 +6,9 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referra
 
 @Component
 class ReferralAccessFilter(
-  private val userTypeChecker: UserTypeChecker,
   private val userAccessScopeMapper: UserAccessScopeMapper,
 ) {
   fun serviceProviderReferrals(referrals: List<Referral>, user: AuthUser): List<Referral> {
-    if (!userTypeChecker.isServiceProviderUser(user)) {
```

Comment:
i think it's best to remove this from here. this method is filtering referrals for SP users, so we don't need to check if this is an SP user or not. it's up to the caller to handle that side of things.